src/test/java/com/digitalasset/refapps/bondissuance/BondIssuanceIT.java

```diff
@@ -52,18 +53,31 @@
               CENTRALBANK_PARTY.getValue(),
               ISSUER_PARTY.getValue(),
               CSD_PARTY.getValue())
-          .setupAppCallback(Main::runBots)
+          .useWallclockTime()
+          .setupAppCallback(
+              Main.runBots(
+                  new PartyAllocator.AllocatedParties(
+                      AGENT_PARTY.getValue(),
+                      BANK1_PARTY.getValue(),
+                      BANK2_PARTY.getValue(),
+                      BANK3_PARTY.getValue(),
+                      CENTRALBANK_PARTY.getValue(),
+                      CSD_PARTY.getValue(),
+                      ISSUER_PARTY.getValue(),
+                      "Operator",
+                      "Regulator"),
+                  getWallclockTimeManager()))
           .build();
 
   @ClassRule public static ExternalResource sandboxClassRule = sandbox.getClassRule();
   @Rule public ExternalResource sandboxRule = sandbox.getRule();
 
   @Test
-  public void testFullWorkflow() throws InvalidProtocolBufferException {
+  public void testFullWorkflow() throws InvalidProtocolBufferException, InterruptedException {
```

Comment:
The code change here does not justify this.
```suggestion
  public void testFullWorkflow() throws InvalidProtocolBufferException {
```